src/components/organisms/Settings/Project/queries.ts

```diff
@@ -138,13 +143,36 @@ export const UPDATE_PROJECT_PUBLIC_DESCRIPTION = gql`
         id
         name
         description
+        imageUrl
         isArchived
         isBasicAuthActive
         basicAuthUsername
         basicAuthPassword
         publicTitle
         publicDescription
+        publicImage
+        alias
+        publishmentStatus
+      }
+    }
+  }
+`;
+
+export const UPDATE_PROJECT_PUBLIC_IMAGE = gql`
```

Comment:
You can combine mutations to update a project into one mutation. 
`updateProject` mutation just ignores unspecified arguments.

```
mutation updateProject($projectId: ID!, $publicTitle: String, $publicImage: Upload, ...) {
```

Reply:
Cool that is much nicer. Updated!